include/core/parameters.h

```diff
@@ -1332,5 +1332,18 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief Return the tensor of entry @p entry_string. If the entry was changed,
+   * then the changed value is returned, otherwise the default value is
```

Comment:
```suggestion
   * @brief Return the tensor of entry @p entry_string. If the entry is specified in the .prm file,
   * then the changed value is returned, otherwise the default value is
```